issue with fonts config and missing 'file-loader' dependency

First of all, cool boilerplate!

I got this error, when I added fonts to the boilerplate

```
ERROR in ./app/styles/fonts/icomoon.svg?jkr1b8
Module parse failed: C:\Users\Tushar\Documents\Projects\temp\react-test\app\styles\fonts\icomoon.svg?jkr1b8 Unexpected token (1:0)
You may need an appropriate loader to handle this file type.
SyntaxError: Unexpected token (1:0)
    at Parser.pp$4.raise (C:\Users\Tushar\Documents\Projects\temp\react-test\node_modules\acorn\dist\acorn.js:2221:15)
    at Parser.pp.unexpected (C:\Users\Tushar\Documents\Projects\temp\react-test\node_modules\acorn\dist\acorn.js:603:10)
    at Parser.pp$3.parseExprAtom (C:\Users\Tushar\Documents\Projects\temp\react-test\node_modules\acorn\dist\acorn.js:1822:12)
    at Parser.pp$3.parseExprSubscripts (C:\Users\Tushar\Documents\Projects\temp\react-test\node_modules\acorn\dist\acorn.js:1715:21)
    at Parser.pp$3.parseMaybeUnary (C:\Users\Tushar\Documents\Projects\temp\react-test\node_modules\acorn\dist\acorn.js:1692:19)
    at Parser.pp$3.parseExprOps (C:\Users\Tushar\Documents\Projects\temp\react-test\node_modules\acorn\dist\acorn.js:1637:21)
    at Parser.pp$3.parseMaybeConditional (C:\Users\Tushar\Documents\Projects\temp\react-test\node_modules\acorn\dist\acorn.js:1620:21)
    at Parser.pp$3.parseMaybeAssign (C:\Users\Tushar\Documents\Projects\temp\react-test\node_modules\acorn\dist\acorn.js:1597:21)
    at Parser.pp$3.parseExpression (C:\Users\Tushar\Documents\Projects\temp\react-test\node_modules\acorn\dist\acorn.js:1573:21)
    at Parser.pp$1.parseStatement (C:\Users\Tushar\Documents\Projects\temp\react-test\node_modules\acorn\dist\acorn.js:727:47)
    at Parser.pp$1.parseTopLevel (C:\Users\Tushar\Documents\Projects\temp\react-test\node_modules\acorn\dist\acorn.js:638:25)
    at Parser.parse (C:\Users\Tushar\Documents\Projects\temp\react-test\node_modules\acorn\dist\acorn.js:516:17)
    at Object.parse (C:\Users\Tushar\Documents\Projects\temp\react-test\node_modules\acorn\dist\acorn.js:3098:39)
    at Parser.parse (C:\Users\Tushar\Documents\Projects\temp\react-test\node_modules\webpack\lib\Parser.js:902:15)
    at DependenciesBlock.<anonymous> (C:\Users\Tushar\Documents\Projects\temp\react-test\node_modules\webpack\lib\NormalModule.js:104:16)
    at DependenciesBlock.onModuleBuild (C:\Users\Tushar\Documents\Projects\temp\react-test\node_modules\webpack-core\lib\NormalModuleMixin.js:310:10)
 @ ./~/css-loader!./~/sass-loader?modules&localIdentName=[name]---[local]---[hash:base64:5]!./app/styles/fonts.scss 6:373-410
```

this was due to webpack loader config for fonts, which I fixed updating the configuration
Here's my new configuration
```
[
  { test: /\.woff(2)?(\?[a-z0-9#=&.]+)?$/, loader: 'url?limit=10000&mimetype=application/font-woff' },
  { test: /\.(ttf|eot|svg)(\?[a-z0-9#=&.]+)?$/, loader: 'file' }
]
```

The old configuration would only support pattern like
- `fontawesome.woff2?v=4.3.1` and not
- `fontawesome.woff2?v=4.3` or `icomoon.svg?jkr1b8#icomoon`

after fixing this I started getting another error

```
ERROR in ./~/css-loader!./~/sass-loader?modules&localIdentName=[name]---[local]---[hash:base64:5]!./app/styles/fonts.scss
Module not found: Error: Cannot resolve module 'file' in C:\Users\Tushar\Documents\Projects\temp\react-test\app\styles
 @ ./~/css-loader!./~/sass-loader?modules&localIdentName=[name]---[local]---[hash:base64:5]!./app/styles/fonts.scss 6:82-119 6:142-179
```

this was due to `file-loader` module was missing

Raising a pull requests for this

Thanks

--- a/package.json
+++ b/package.json
@@ -17,6 +17,7 @@
   "author": "Jean-Pierre Sierens",
   "license": "MIT",
   "dependencies": {
+    "file-loader": "^0.9.0",
     "react": "^15.0.2",
     "react-dom": "^15.0.2",
     "react-hot-loader": "^3.0.0-beta.2",

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -57,8 +57,8 @@ module.exports = {
                 test: /\.scss$/,
                 loader: 'style!css!sass?modules&localIdentName=[name]---[local]---[hash:base64:5]'
             },
-            { test: /\.woff(2)?(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: "url-loader?limit=10000&minetype=application/font-woff" },
-            { test: /\.(ttf|eot|svg)(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: "file-loader" }
+            { test: /\.woff(2)?(\?[a-z0-9#=&.]+)?$/, loader: 'url?limit=10000&mimetype=application/font-woff' },
+            { test: /\.(ttf|eot|svg)(\?[a-z0-9#=&.]+)?$/, loader: 'file' }
         ]
     }
 };

--- a/webpack.production.config.js
+++ b/webpack.production.config.js
@@ -82,12 +82,12 @@ module.exports = {
             // we extract the styles into their own .css file instead of having
             // them inside the js.
             loader: ExtractTextPlugin.extract('style', 'css!postcss!sass')
-        },{
-            test: /\.woff(2)?(\?v=[0-9]\.[0-9]\.[0-9])?$/,
-            loader: "url-loader?limit=10000&minetype=application/font-woff"
-        },{
-            test: /\.(ttf|eot|svg)(\?v=[0-9]\.[0-9]\.[0-9])?$/,
-            loader: "file-loader"
+        }, {
+            test: /\.woff(2)?(\?[a-z0-9#=&.]+)?$/,
+            loader: 'url?limit=10000&mimetype=application/font-woff'
+        }, {
+            test: /\.(ttf|eot|svg)(\?[a-z0-9#=&.]+)?$/,
+            loader: 'file'
         }]
     },
     postcss: [

